src/GuiBase/Viewer/WindowQt.hpp

```diff
@@ -0,0 +1,77 @@
+// This file si picked and slightly adapted from
+// globjects/source/examples/qtexample/WindowQt.h
+// so that update in globjects is easy
+// Radium Viewer derive this class
+
+#pragma once
+#include <GuiBase/RaGuiBase.hpp>
+
+#include <QScopedPointer>
+#include <QWindow>
+
+#include <glbinding/ProcAddress.h>
+
+#include <atomic>
+
+class QSurfaceFormat;
+class QOpenGLContext;
+
+namespace Ra {
+namespace Gui {
+
+class RA_GUIBASE_API WindowQt : public QWindow {
```

Comment:
Could you mention this class comes from globjects examples ?
```suggestion
/*!
 * \brief Base class for OpenGL widgets, compatble with Qt and globjects/glbindings.
 * \note This class has been extracted from globject example _add-url-here_
 */
class RA_GUIBASE_API WindowQt : public QWindow {
```